### ISSUE TITLE
Multiple Master Databases for one Slave Database 

### ISSUE THREAD

**Issue by nils-schween** [joostkremers/ebib#161]
Hi everyone,

I was wondering if it is possible that a slave database has multiple master databases. My use case is the following: I have got a folder "Library" and inside there are, for example, folders like "Books" or "Paper". Every subcategory has its own bib file. For the above example, these would be "Library/Books/books.bib" and "Library/Paper/paber.bib". 

Now, I am creating a new LaTeX project and I would love it to have its own bib file. But of course, in the bib file, I would like to have entries belonging to different categories like "Books" and "Papers". Thus, it would be perfect if the bib file of my new latex project had several master files.

Is this possible?
Thanks,
Nils

**Comment by joostkremers** [comment]
No, this is currently not possible. I'm afraid you'll have to make do with Ebib's export function (by default bound to `x` in the index buffer), or copy and paste entries. (The command to copy an entry, `ebib-copy-entry`, is not bound by default, the command to paste an entry is by default bound to `y`.)

I considered this option when I first implemented master/slave databases and decide it wasn't worth the trouble. I could go through the code and see how much effort it would take to change this, but I can't promise anything.

**Comment by nils-schween** [comment]
Thanks a lot! I'll bind the ```ebib-copy-entry```  function to a convenient key and this should cover my use case.

**Comment by nils-schween** [comment]
@joostkremers I was wondering if you could reconsider implementing multiple main databases for a dependent database. Probably it is difficult to implement, but if it was not, it would great to have this feature. And to me it seems that it fits well into the design of ebib being able to handle multiple databases.

Thanks for reconsidering,
Nils

**Comment by joostkremers** [comment]
Hi, I still can't promise anything, because the change would be rather substantial.
